psycop/common/feature_generation/sequences/timeseries_windower/types/abstract_polars_dataframe.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import asdict, dataclass
+from typing import Any
+
+import polars as pl
+
+
+@dataclass(frozen=True)
+class ColumnBundle:
+    entity_id: str = "entity_id"
+
+
+class PolarsDataframeBundle:
+    def __init__(
+        self,
+        df: pl.LazyFrame,
+        cols: ColumnBundle,
+        validate_cols_exist_on_init: bool = True,
```

Comment:
Why would this be optional? (is it due to being lazy?)

Reply:
Makes for easier testing, e.g. in `test_should_be_frozen`.

Reply:
Virker som et funky trade-off? Hvorfor ikke bare have rigtige kolonner i testen?